index.js

```diff
@@ -118,6 +118,16 @@ class SimplePicker extends Component {
 				modalVisible: props.modalVisible,
 			});
 		}
+
+		// Update current selected option if initialOptionIndex is updated
+		if (
+			props.options
+			&& props.initialOptionIndex
+			&& (props.options[props.initialOptionIndex] !== this.state.selectedOption)
```

Comment:
Correct me if I am wrong, but doesn't this reset the selected option if the props change? If the user changes the selectedOption this will return true and it will be resetted to the intial option